packages/core/directives/map.ts

```diff
@@ -456,9 +465,9 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
   private _handleMapCenterChange() {
     const s = this._mapsWrapper.subscribeToMapEvent<void>('center_changed').subscribe(() => {
       this._mapsWrapper.getCenter().then((center: LatLng) => {
-        this.latitude = center.lat();
-        this.longitude = center.lng();
-        this.centerChange.emit(<LatLngLiteral>{lat: this.latitude, lng: this.longitude});
+        this._mapLatitude = center.lat();
+        this._mapLongitude = center.lng();
```

Comment:
Using LatLng type means you can then use the equals method for accurate comparison.